test-app/app/src/MyComponent.js

```diff
@@ -118,9 +189,44 @@ export default () => (
               drizzleState={drizzleState}
               contract="ComplexStorage"
               method="singleDD"
+              render={displayData => {
+                var i = 0;
+                const displayObjectProps = [];
+
+                Object.keys(displayData).forEach(key => {
+                  if (i !== key) {
```

Comment:
Hmm I'm not really sure what's going on here. Was the intention to filter out the keys 0, 1, 2 (since they give the same values as keys deviceBrand, deviceYear, batteryWearLevel)?

Because I get the following output:

```
Single Device Data:
Element 0 has key: 0
and value: deviceBrand3
Element 1 has key: 1
and value: deviceYear3
Element 2 has key: 2
and value: wearLevel3
Element 3 has key: deviceBrand
and value: deviceBrand3
Element 4 has key: deviceYear
and value: deviceYear3
Element 5 has key: batteryWearLevel
and value: wearLevel3
```

Reply:
Oh, when I think again, eslint complained about `!=` and wanted to have `!==`, so probably eslint was wrong? In your code, you have `!=`. I'll change it back…

Reply:
Yes, eslint complains. I changed it back to `!=`.